src/ziggurat/streams.clj

```diff
@@ -114,7 +114,7 @@
   message)
 
 (defn store-supplier-builder []
-  (KeyValueStoreBuilder. (RocksDbKeyValueBytesStoreSupplier. "state-store")
+  (KeyValueStoreBuilder. (RocksDbKeyValueBytesStoreSupplier. "state-store" false)
```

Comment:
why did we add a false here? Is this a breaking change in the version upgrade?
Asking as I did not find anything in the upgrade guide - https://kafka.apache.org/23/documentation/streams/upgrade-guide#streams_api_changes_221

Reply:
yep, the constructor of RocksDbKeyValueBytesStoreSupplier changed from 2.2.0